lib/mixlib/install/options.rb

```diff
@@ -213,8 +218,15 @@ def validate_platform_options
         end
       end
 
-      def map_windows_desktop_versions!
-        options[:platform_version] = Util.map_windows_desktop_version(platform_version)
+      def map_windows_versions!
+        return unless for_windows?
+
+        # Store original platform version for nano requests
+        if SUPPORTED_WINDOWS_NANO_VERSIONS.include?(options[:platform_version])
```

Comment:
Feels like we should always set `@original_platform_version`, I would remove this guard check.